src/content/en/updates/2017/01/nic56.md

```diff
@@ -32,16 +32,22 @@ I’m Pete LePage, let’s dive in and see what’s new for developers in Chrome
 
 ## Web Bluetooth API {: #webbluetooth }
 
-Until now, the ability to interact with bluetooth devices has only been
-possible for native apps. With Chrome 56, your web apps can communicate
-with nearby Bluetooth Low Energy devices in a secure and private manner
-using the [Web Bluetooth API](//webbluetoothcg.github.io/web-bluetooth/). 
+Until now, users who have a bluetooth devices were forced to install
+native apps to communicate with them. With Chrome 56, users can select
```

Comment:
Either:
* a bluetooth device ... communicate with it
*bluetooth devices ... communicate with them